azure-pipelines/azure-pipelines.yml

```diff
@@ -521,6 +523,17 @@ stages:
           --build-arg PROJECT_ROOT="$(System.DefaultWorkingDirectory)" $(System.DefaultWorkingDirectory) \
           -t edgehub/deviceshifu-http-http:$(release_tag) --push
         displayName: build edgehub/deviceshifu-http-http
+      - task: Docker@2
+        displayName: Login to DockerHub
+        inputs:
+          command: login
+          containerRegistry: cn-dockerhub-connection
```

Comment:
`cn-dockerhub-connection` => `cn-docker-connection` , same for all other occurrences